pkg/kepctl/kepctl.go

```diff
@@ -373,6 +374,39 @@ func (c *Client) loadKEPFromYaml(kepPath string) (*api.Proposal, error) {
 		return nil, fmt.Errorf("unable to load KEP metadata: %s", err)
 	}
 	p.Name = filepath.Base(filepath.Dir(kepPath))
+
+	// Read the PRR approval file and add any listed PRR approvers in there
+	// to the PRR approvers list in the KEP. this is a hack while we transition
+	// away from PRR approvers listed in kep.yaml
+	prrPath := filepath.Dir(kepPath)
+	prrPath = filepath.Dir(prrPath)
+	sig := filepath.Base(prrPath)
+	prrPath = filepath.Join(filepath.Dir(prrPath),
+		"prod-readiness",
+		sig,
+		p.Number+".yaml")
+	prrFile, err := os.Open(prrPath)
+	if !os.IsNotExist(err) {
+		if err != nil {
+			return nil, fmt.Errorf("could not open file %s: %v\n", prrPath, err)
```

Comment:
the flow used to return without error when the prr yaml wasn't present.  Seems like we still want to allow that case, right?

Reply:
it still does. if !os.IsNotExists line above

Reply:
different style.

I'm used to 
```go
if os.IsNotExist(err){
  return existing
}
if err != nil{
  fail
}
code parsing new values.  
```

Reply:
yeah, that's better.

Reply:
that's what I would normally do, don't know why i did it this way